src/main/java/com/dungeoncrawler/Javiarenka/shop/Shop.java

```diff
@@ -0,0 +1,58 @@
+package com.dungeoncrawler.Javiarenka.shop;
+
+import com.dungeoncrawler.Javiarenka.character.Hero;
+import com.dungeoncrawler.Javiarenka.character.HeroClass;
+import com.dungeoncrawler.Javiarenka.equipment.Armor;
+import com.dungeoncrawler.Javiarenka.equipment.Weapon;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Shop {
+
+    private static List<Armor> allAvailableArmor = new ArrayList<>(Arrays.asList(null, null, null, null, null)); // todo
+    private static List<Weapon> allAvailableWeapon = new ArrayList<>(Arrays.asList(null, null, null, null, null)); // todo
+
+    public static List<Armor> getNewArmor(Object object) {
```

Comment:
O ile metodę validate rozumiem i mega się mi ona podoba, o tyle nie rozumiem do końca czemu nasz getNewArmor() przyjmuje obiekt zamiast od razu przyjmować Herosa.

Reply:
Zmieniłam to w docelowej metodzie getNewEquipment() ;)